src/index.html

```diff
@@ -17,16 +17,20 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form action="post">
+      <input
+        class="input big-input"
+        type="text"
+        data-qa="big"
+        placeholder="Try “Los Angeles“"
+      />
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+      <input
+        class="input small-input"
```

Comment:
Similar to the previous comment, the class name 'small-input' should be reconsidered to reflect the content or function rather than the style.